conviva/src/main/java/com/bitmovin/analytics/conviva/ContentMetadataBuilder.java

```diff
@@ -133,6 +131,22 @@ public Map<String, String> getCustom() {
         return customs;
     }
 
+    public void setAdditionalStandardTags(Map<String, Object> newValue) {
+        metadata.setAdditionalStandardTags(newValue);
+    }
+
+    public Map<String, Object> getAdditionalStandardTags() {
```

Comment:
This function is is basically the same as for [getCustom()](https://github.com/bitmovin/bitmovin-player-android-analytics-conviva/pull/56/files#diff-d3929f8074b1dbb1e079905a85028bc5881c094de071b809e181612a7fb8de04R122)